packages/plugin-ext/src/plugin/command-registry.ts

```diff
@@ -123,3 +130,51 @@ export class CommandRegistryImpl implements CommandRegistryExt {
         this.argumentProcessors.push(processor);
     }
 }
+
+export class CommandsConverter {
```

Comment:
make sure to file a CQ for all copied code, it's copied and modified from here: https://github.com/TypeFox/vscode/blob/70b8db24a37fafc77247de7f7cb5bb0195120ed0/src/vs/workbench/api/common/extHostCommands.ts#L221-L263

please add a comment in the code and MS copyright under ours

cc @theia-ide/ip-help

Reply:
Added a copyright and a comment, opened a CQ: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=20680

Reply:
@akosyakov The CQ has been approved!

Reply:
cool, thank you!